src/common/column_matrix.h

```diff
@@ -71,7 +71,6 @@ class ColumnMatrix {
                 double  sparse_threshold) {
     const auto nfeature = static_cast<bst_uint>(gmat.cut.row_ptr.size() - 1);
     const size_t nrow = gmat.row_ptr.size() - 1;
-
```

Comment:
There seems to be a few unnecessary formatting changes in this PR. It makes it harder to review as we have to search through these carefully to see if anything is actually changed, as well as adding unnecessary history to these files.